src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
I dont see how that makes any difference... They are the same colours, the order they are displayed makes no difference here... Or if you really wanted to make them different you'd just add one to the colour being picked on that method